platform/packaging/build/faiss/Dockerfile

```diff
@@ -123,18 +141,21 @@ RUN cmake --fresh \
       -D CMAKE_BUILD_TYPE:STRING=Release \
       -W no-dev
 
+
 ###
 #
 # Build faiss, swigfaiss, swigfaiss_avx2, and then install the python callback
 # Next, build a faiss wheel from the compiled source.
 
-RUN make -j$(nproc) -C _build faiss
-RUN make -j$(nproc) -C _build swigfaiss
-RUN make -j$(nproc) -C _build swigfaiss_avx2
-RUN make -j$(nproc) -C _build install
+WORKDIR "/workspace/build/faiss-${FAISS_VERSION}"
+RUN make --jobs=$(nproc) --directory="_build" faiss
+RUN make --jobs=$(nproc) --directory="_build" swigfaiss
+RUN make --jobs=$(nproc) --directory="_build" swigfaiss_avx2
+RUN make --jobs=$(nproc) --directory="_build" install
```

Comment:
For some reason, Make isn't happy with the non-existence of `_build` here.

Reply:
agree. The reason is `WORKDIR` is not set correclty, and the absolute path was not provided.